code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -225,17 +225,22 @@
 
 /obj/item/clothing/head/helmet/space/plasmaman/atmospherics
 	name = "atmospherics plasma envirosuit helmet"
-	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue."
+	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue. Has thermal shielding." // Added the thermal shielding clarification
 	icon_state = "atmos_envirohelm"
 	inhand_icon_state = "atmos_envirohelm"
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 100, RAD = 10, FIRE = 100, ACID = 75)
+	heat_protection = HEAD
```

Comment:
This line isn't needed, the normal space helmets have it already.
```suggestion
```

Reply:
Ah, missed that. Thank you for pointing it out